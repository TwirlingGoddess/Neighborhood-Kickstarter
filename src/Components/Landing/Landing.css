.landing-page {
  margin: auto;
  width: 100vw;
  height: 100vh;
  /*overflow: scroll;*/
}