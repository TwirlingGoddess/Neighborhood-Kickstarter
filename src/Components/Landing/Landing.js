import React, { Component } from 'react';
import Search from'../Search/Search';
import { ProjectsContainer } from '../ProjectsContainer/ProjectsContainer';
import './Landing.css'

class Landing extends Component {
  constructor() {
    super()
    this.state = {
      currentUser: ''
    }
  }
  
  render() {
    return(
      <div className='landing-page'>
        <h1>Landing</h1>
        <Search />
        <ProjectsContainer />
      </div>
    )
  }
}

export default Landing