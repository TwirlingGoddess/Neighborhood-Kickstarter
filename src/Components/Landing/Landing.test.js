import React from 'react';
import Landing from './Landing';
import { shallow } from 'enzyme';

describe('Landing', () => {
  let wrapper;
  let mockCurrentUser = {};
  it('should match the snapshot', () => {
    wrapper = shallow(<Landing currentUser={mockCurrentUser}/>);
    expect(wrapper).toMatchSnapshot();
  });
}); 