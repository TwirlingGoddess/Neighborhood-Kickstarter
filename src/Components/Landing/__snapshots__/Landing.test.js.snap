// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Landing
    currentUser={Object {}}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <StateManager
          className="select-input"
          defaultInputValue=""
          defaultMenuIsOpen={false}
          defaultValue={null}
          onChange={[Function]}
          options={Array []}
          placeholder="Search a neighborhhod for projects"
          value=""
        />,
        <ProjectsContainer
          allUsers={Array []}
          projects={Array []}
        />,
      ],
      "className": "landing-page",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "className": "select-input",
          "defaultInputValue": "",
          "defaultMenuIsOpen": false,
          "defaultValue": null,
          "onChange": [Function],
          "options": Array [],
          "placeholder": "Search a neighborhhod for projects",
          "value": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "allUsers": Array [],
          "projects": Array [],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <StateManager
            className="select-input"
            defaultInputValue=""
            defaultMenuIsOpen={false}
            defaultValue={null}
            onChange={[Function]}
            options={Array []}
            placeholder="Search a neighborhhod for projects"
            value=""
          />,
          <ProjectsContainer
            allUsers={Array []}
            projects={Array []}
          />,
        ],
        "className": "landing-page",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "className": "select-input",
            "defaultInputValue": "",
            "defaultMenuIsOpen": false,
            "defaultValue": null,
            "onChange": [Function],
            "options": Array [],
            "placeholder": "Search a neighborhhod for projects",
            "value": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "allUsers": Array [],
            "projects": Array [],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
